seleniumbase/core/browser_launcher.py

```diff
@@ -1679,6 +1693,7 @@ def get_local_driver(
                 headless,
                 locale_code,
                 proxy_string,
+                proxy_bypass_list,
                 proxy_auth,
                 proxy_user,
                 proxy_pass,
```

Comment:
@grahamhar Looks like you still have one or two more places where you have to fix the ordering

Reply:
fixed